### DISCUSSION TITLE
video download coapp

### DISCUSSION THREAD

**Discussion by mocos46** [aclap-dev/video-downloadhelper#121] [Help and Q&A]
### Your question

Dear Paul, I found a new download vdhcoapp-mac13-x86_64-installer.pkg for old versions of OSx (14, Mojave), but it doesn't install. The installer hangs forever while writing scripts (or something like that). Tried many times, and often it blocks the system, selflaunching itself also after a forced out, even with application Monitor, and had to do a hard reboot. Thanks for your efforts

**Comment by paulrouget** [comment]
Can you install the DMG, extract it anywhere you want, and then run it?

**Reply by mocos46** [reply]
Sorry, I didn't find a dmg. I found only a .pkg for mac osx 13 e 14...

**Reply by paulrouget** [reply]
[vdhcoapp-mac13-x86_64.dmg](https://github.com/aclap-dev/vdhcoapp/releases/download/v2.0.17/vdhcoapp-mac13-x86_64.dmg)

**Comment by MrLonkers** [comment]
I am having the exact same problem. It always hangs, no matter which version of the installer I use. Even after a Force Quit I have to Hard Shutdown, including the one above. After restarting and trying to run it again, it says please wait while previous install finishes, and hangs again. Mac os 12.7.4
